docs/rfcs/multi-schedules.md

```diff
@@ -0,0 +1,80 @@
+# Summary
+
+The Multi Schedule function allows you to register multiple schedules at once.
+It is managed together like a staged "progressive rollout".
+
+## Function
+### Create
+It is possible to specify ON or OFF for multiple dates at the same time.
+* Validation
+  * Dates must be set in ascending order
+  	* Error ex： 1/1[ON] 1/3[OFF] 1/2[ON]
+  * Validation of ON/OFF order is not performed.
+  	* OK ex： 1/1[ON] 1/2[ON] 1/3[OFF]
+
+### Update
+Settings for future dates can be updated or deleted.
+* Validation
+  * Schedules that have already been completed cannot be updated
+  	* ex: Setting was 1/1 0:00 [ON]  1/2 0:00 [OFF] 1/3 0:00 [ON] .\
+     As of 1/2 1:00, updates are possible only on 1/3 0:00 ON
+
+### Delete
+It is possible to delete uncompleted schedules.
+* Validation
+  * Schedules that have already been completed cannot be deleted.
+
+### Stop
+It is possible to stop a schedule that has already started.
+* Validation
+  * Schedules that have already been completed cannot be stopped.
```

Comment:
Do you think that stop is necessary?
We can just delete a schedule instead of stopping it.

Reply:
On the operations page, there are two tabs: one for active operations and the other for finished or stopped operations.

When using multiple schedules, the user might want to stop the operation to keep the history of the other schedules executed earlier.
The Progressive Rollout has the same behavior.